modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/gatewayPolicy/GatewayPolicyTestCase.java

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Comment:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Reply:
Added via: https://github.com/wso2/product-apim/pull/13317/commits/f1f32cdea86d8b9d82d90ecae572779401fbe05b

Thank you.